### ISSUE TITLE
xmerl_dsig fails due to mismatched SubjectPublicKeyInfo record format

### ISSUE THREAD

**Issue by chefsalim** [arekinath/esaml#19]
Using Erlang 18.

xmerl_dsig line 204 is:

```
       {_, KeyBin} = Cert#'Certificate'.tbsCertificate#'TBSCertificate'.subjectPublicKeyInfo#'SubjectPublicKeyInfo'.subjectPublicKey,
```

This does not match the record definition for SubjectPublicKeyInfo from the erlang docs site:

```
#'SubjectPublicKeyInfo'{
      algorithm,       % #AlgorithmIdentifier{} 
      subjectPublicKey % binary() 
     }.
```

so it’s expecting a `{any(), binary()}` and getting a `binary()`

**Comment by arekinath** [comment]
This should have been fixed by 446dc02cf40213ef901d5887a434702d24cf1b58 on the `develop` branch. It hasn't been merged to master yet -- I'd like to wait for #19 and #15 if possible before releasing.

**Comment by chefsalim** [comment]
Cool, thanks
